src/assets/scripts/math/circle.js

```diff
@@ -64,3 +64,27 @@ export const fix_angle = (radians) => {
 
     return radians;
 };
+
+/**
+ * normalize angles to within 0° - 360°
+ * @param  {number} degrees an angle
+ * @return {number}         an angle within [0,360]
+ */
+export const degrees_normalize = (degrees) => {
+    if (degrees < 0) {
+        return 360 + (degrees % 360);
+    }
+    else return degrees % 360;
+};
+
+/**
+ * normalize angles to within 0 - 2π
+ * @param  {number} radians an angle
+ * @return {number}         an angle within [0,2π]
+ */
+export const radians_normalize = (radians) => {
+    if (radians < 0) {
+        return (Math.PI*2) + (radians % (Math.PI*2));
+    }
+    else return radians % (Math.PI*2);
```

Comment:
With `tau` being in the same file as an *export*, and not as a variable, my attempts to call it were coming up undefined... internet said to either import the file to itself or to access it via `module.exports.tau()`, but these are being exported via the `export` keyword directly, and not as `module.exports`, so I figured it wouldn't work to do that either (admittedly I didn't try). I fully intended to use tau elsewhere though, just had trouble using it in the same file.

And yeah it can be flopped, I care not. haha as for the oneline if/else, that's fine by me. I'll remove it.